domain-cc/cc-app/src/python_src/api.py

```diff
@@ -124,15 +125,19 @@ def log_contention_stats(
     else:
         log_contention_type = contention.contention_type.lower()
 
+    is_multi_contention = len(claim.contentions) > 1
+
     log_as_json(
         {
+            "claim_id": sanitize_log(claim.claim_id),
```

Comment:
can you switch `"claim_id"` to `"vagov_claim_id"` so that it doesn't get marked as duplicated on current datadog dashboard?

Reply:
![Screenshot 2024-06-11 at 11 15 03 AM](https://github.com/department-of-veterans-affairs/abd-vro/assets/17188013/ea7f1a0f-a898-4ef6-8f11-ad8fd863f027)